src/actions/index.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+export const FETCH_PRODUCTS = "fetch_products";
+export const SEARCH_PRODUCTS = "search_products";
+
+const ROOT_URL = "http://localhost:8000/products";
+
+//This api call fetches all the products through the "/products" route 
+export function fetchProducts() {
+    const request = axios.get(ROOT_URL);
+
+    return {
+        type: FETCH_PRODUCTS,
+        payload: request
+    }
+}
+
+//This api call fetches all categories based on the query the user searches for in search bar 
+export function searchProducts(values) {
+    const request = axios.get(`${ROOT_URL}?page=1&category=${values}`)
```

Comment:
Aren't there other times when we need to fetch products by `page` and `category` other than the "search"?